### ISSUE TITLE
Can it be used in sd1.5 and can it be combined with other acceleration methods such as ByteDance/Hyper SD

### ISSUE THREAD

**Issue by libai-lab** [A-suozhang/MixDQ#1]
Can it be used in sd1.5 and can it be combined with other acceleration methods such as ByteDance/Hyper SD

**Comment by A-suozhang** [comment]
MixDQ supports SD1.5. By using the lcm_lora.yaml file, you can conduct quantization for SD1.50like models (Dreamlike) with LCM-lora. Our quantization code is independent of the timestep-wise acceleration method. By substituting the sdxl-turbo model ID in the config, it is compatible with HyperSD.

**Comment by greasebig** [comment]
Can it be used directly in sd1.5 or sdxl. What i mean is using W8A8 to accelerate normal 20 steps inference, without lcm_lora or sdxl turbo.

**Comment by A-suozhang** [comment]
Yes, it could be directly used. Just follow the example sdxl.yaml in our configs. For Sd1.5 model, you could remove the lora-related configs in `lcm_lora.yaml`, it`s compatible with standard Sd1.5.

**Comment by greasebig** [comment]
if I want to use normal sdxl 20steps inference in the pipeline from https://huggingface.co/nics-efc/MixDQ/tree/main, what I should do ?

**Comment by greasebig** [comment]
pipeline from https://huggingface.co/nics-efc/MixDQ/tree/main, seems to only compatible with  lcm_lora and sdxl turbo.

**Comment by greasebig** [comment]
seems like i need to Generate Calibration Data and Post Training Quantization (PTQ) Process.
it may cause very long time.
do you have the quant parameters ckpt.pth that can be used directly in sdxl?
